### PR TITLE
Add a description of how to get a list of plugins, get meta of a plugin

### PR DESCRIPTION
**Pull request by skabashnyuk**
Add a description of how to get a list of plugins, get meta of a plugin.

### What does this PR do?
Reference @garagatyi comment https://github.com/eclipse/che/issues/10834#issuecomment-414249814

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,67 @@ You can deploy Che plugin registry on Openshift with command.
              -p IMAGE_TAG="latest" \
              -p PULL_POLICY="IfNotPresent"
 ```
+## Docker
+```
+docker run -it  -p 8080:8080 eclipse/che-plugin-registry
```
Comment (benoitf) [inline README.md]: maybe add `--rm`

File: README.md [hunk 2]
```diff
@@ -21,6 +21,67 @@ You can deploy Che plugin registry on Openshift with command.
...
+```
+
+## Get index list of all plugins
+Example:
+```
+curl  "http://localhost:8080/index.json"
```
Comment (benoitf) [inline README.md]: could it be directly on / ?

File: README.md [hunk 3]
```diff
@@ -21,6 +21,67 @@ You can deploy Che plugin registry on Openshift with command.
...
+    "links": {
+      "self": "/plugins/che-dummy-plugin/0.0.1/meta.yaml"
+    }
+  }
+]
+```
+## Get meta.yaml of a plugin
+Example:
+```
+curl  "http://localhost:8080/plugins/org.eclipse.che.editor.theia/1.0.0/meta.yaml"
```
Comment (benoitf) [inline README.md]: could it be http://localhost:8080/plugins/org.eclipse.che.editor.theia/1.0.0 ?

`/plugins/<id>/<version>`

Comment (skabashnyuk) [reply README.md]: Not for now. In general   - don't know. Would you like to request this as a new feature?

File: README.md (hunk 2 above)
Comment (skabashnyuk) [reply README.md]: Not for now. In general  - don't know. Would you like to request this as a new feature?

File: README.md (hunk 1 above)
Comment (skabashnyuk) [reply README.md]: like this 
```
docker run -it --rm  -p 8080:8080 eclipse/che-plugin-registry
``` 
?

Comment (benoitf) [reply README.md]: yes

File: README.md (hunk 2 above)
Comment (benoitf) [reply README.md]: https://github.com/eclipse/che-plugin-registry/issues/26

**Pull request merged**